IanGRap/Squish
# Bugfix: #48 (Infinite Jump)

Contributors: Austin

Goals: 
- Fix the bug were players could jump forever, even if they don't touch the ground
- Allow players to jump on each other's heads

What Changed:
- In PlayerGrounded.cs, I added a filter to the Physics Overlap Sphere, so that it does not pick up the player it is attached to
- I removed the "Player" layer on all child objects of the player prefab because the Ground Collider was picking up those objects as well

How to Test:
- Load up any functioning scene (I used garbage compactor) and spam the jump button. You should not be able to jump infinitely
- Try jumping on someone else's head.
